x-pack/plugins/alerting/public/pages/maintenance_windows/translations.ts

```diff
@@ -446,3 +446,7 @@ export const SAVE_MAINTENANCE_WINDOW = i18n.translate(
     defaultMessage: 'Save maintenance window',
   }
 );
+
+export const UPCOMING = i18n.translate('xpack.alerting.maintenanceWindows.upcoming', {
+  defaultMessage: 'UPCOMING',
```

Comment:
Unless there's a strong reason for capitalizing this, I prefer the less "yell-y":
```suggestion
  defaultMessage: 'Upcoming',
```